hermes/src/aggregate.rs

```diff
@@ -152,10 +152,11 @@ pub enum Update {
 
 #[derive(Debug, PartialEq)]
 pub struct PriceFeedUpdate {
-    pub price_feed:  PriceFeed,
-    pub slot:        Option<Slot>,
-    pub received_at: Option<UnixTimestamp>,
-    pub update_data: Option<Vec<u8>>,
+    pub price_feed:        PriceFeed,
+    pub slot:              Option<Slot>,
+    pub received_at:       Option<UnixTimestamp>,
+    pub update_data:       Option<Vec<u8>>,
+    pub prev_publish_time: Option<UnixTimestamp>,
```

Comment:
I realise we can't change it at this point but It's a shame this isn't `last_publish_time` like other `last_` fields we have in our codebases.

Reply:
i think here `prev` is more clear than `last`.